modular_skyrat/modules/gladiator/code/game/objects/items/gladiator_items.dm

```diff
@@ -19,23 +28,100 @@
 	if(.)
 		incomingchance.block_chance -= bonus_value
 
-/obj/item/claymore/agateram //it just works
-	name = "ancient blade agateram"
-	desc = "A millenia-old blade made from a material that you can't even begin to fathom. It flows with the power of the Marked One who once held it." //That thing was too big to be called a sword. Too big, too thick, too heavy, and too rough, it was more like a large hunk of iron.
-	icon = 'modular_skyrat/master_files/icons/obj/agateram.dmi'
-	icon_state = "demonsword"
-	inhand_icon_state = "demonsword"
-	lefthand_file = 'modular_skyrat/master_files/icons/mob/agateraminhandsleft.dmi'
-	righthand_file = 'modular_skyrat/master_files/icons/mob/agateraminhandsright.dmi'
+/obj/item/clothing/neck/warrior_cape
+	name = "\proper cloak of the marked one"
+	desc = "A cloak worn by those that have faced death in the eyes and prevailed."
+	icon = 'modular_skyrat/modules/gladiator/icons/berserk_icons.dmi'
+	worn_icon = 'modular_skyrat/modules/gladiator/icons/berserk_suit.dmi'
+	icon_state = "berk_cape"
+	inhand_icon_state = "" //lul
+	uses_advanced_reskins = FALSE
+	resistance_flags = INDESTRUCTIBLE
+
+/obj/item/clothing/neck/warrior_cape/examine()
+	. = ..()
+	. += span_warning("Struggle against the tide, no matter how strong it may be.")
+
+/obj/item/clothing/suit/hooded/berserker/gatsu
+	name = "\proper berserker armor"
+	desc = "A suit of ancient body armor imbued with potent spiritual magnetism, capable of massively boosting a wearer's close combat skills at the cost of ravaging their mind and overexerting their body."
+	icon_state = "berk_suit"
+	icon = 'modular_skyrat/modules/gladiator/icons/berserk_icons.dmi'
+	worn_icon = 'modular_skyrat/modules/gladiator/icons/berserk_suit.dmi'
+	hoodtype = /obj/item/clothing/head/hooded/berserker/gatsu
+	armor = list(MELEE = 45, BULLET = 40, LASER = 30, ENERGY = 30, BOMB = 80, BIO = 100, FIRE = 100, ACID = 100)
+	resistance_flags = INDESTRUCTIBLE
+
+/obj/item/clothing/suit/hooded/berserker/gatsu/examine()
+	. = ..()
+	. += span_warning("Berserk mode requires the suit's helmet to be equipped, and can only be charged by taking damage.")
+
+/obj/item/clothing/head/hooded/berserker/gatsu
+	name = "\proper berserker helmet"
+	desc = "A uniquely styled helmet with ghastly red eyes that seals it's user inside."
+	icon_state = "berk_helm"
+	icon = 'modular_skyrat/modules/gladiator/icons/berserk_icons.dmi'
+	worn_icon = 'modular_skyrat/modules/gladiator/icons/berserk_suit.dmi'
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDESNOUT
+	armor = list(MELEE = 45, BULLET = 40, LASER = 30, ENERGY = 30, BOMB = 80, BIO = 100, FIRE = 100, ACID = 100)
```

Comment:
```suggestion
	armor = list(MELEE = 40, BULLET = 40, LASER = 20, ENERGY = 25, BOMB = 70, BIO = 100, FIRE = 100, ACID = 100)
```
These values are a bit much

Reply:
yeah, your changes seem reasonable, i've been testing this stuff and was gonna change it up a bit